src/lib/THERMAL/thermal.cpp

```diff
@@ -17,18 +20,32 @@ uint8_t thermal_threshold_data[] = {
 };
 
 int thermal_status = THERMAL_STATUS_FAIL;
+static bool use_internal_sensor = false;
```

Comment:
Unused variable (write-only). Remove and please also change `thermal_status` to `static Thermal_Status_t thermal_status`.

Reply:
Done, plus some other bonus cleanups